kotest-framework/kotest-framework-engine/src/commonMain/kotlin/io/kotest/engine/spec/SpecExecutor.kt

```diff
@@ -47,6 +48,7 @@ internal class SpecExecutor(
 
    private suspend fun executeInDelegate(spec: Spec): Result<Map<TestCase, TestResult>> {
       return try {
+         @Suppress("DEPRECATION")
```

Comment:
Nit:

You did this at the beggining of the file on other files. Now we have 3 of these in the same file

Reply:
I didn't want to just blanket use it here.